engine/table/src/main/java/io/deephaven/engine/table/impl/by/PartitionByChunkedOperator.java

```diff
@@ -58,11 +53,19 @@ public final class PartitionByChunkedOperator implements IterativeChunkedAggrega
     // region nonexistent table sentinels
 
     /**
-     * Sentinel value for the row set belonging to a table that was never created because either the result table was no
-     * longer live or the aggregation update listener was no longer live. Should be used for assignment and reference
-     * equality tests, only.
+     * Sentinel value for the row set builders belonging to a table that was never created because either the result
+     * table was no longer live or the aggregation update listener was no longer live. Should be used for assignment and
+     * reference equality tests, only.
+     */
+    private static final RowSetBuilderRandom NONEXISTENT_TABLE_ROW_SET_BUILDER_RANDOM = RowSetFactory.builderRandom();
+    private static final RowSetBuilderSequential NONEXISTENT_TABLE_ROW_SET_BUILDER_SEQUENTIAL =
```

Comment:
`NONEXISTENT_TABLE_ROW_SET_BUILDER_SEQUENTIAL` is never assigned and not needed.  Can remove it and all tests for it.